fastlane/Scanfile

```diff
@@ -3,4 +3,6 @@ project("Client.xcodeproj")
 device("iPhone 14")
 output_types("junit")
 output_directory("test_output/xml")
+cloned_source_packages_path("SourcePackages")
```

Comment:
Just wondering, shouldn't this folder be added to `.gitignore`?

Reply:
It can be yeah 👌 - although this `SourcePackage` lives in the CI context only